src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure {
+
+    public double area;
+    public String color;
+
+    public Figure(double area, String color) {
+        this.area = area;
+        this.color = color;
+    }
+
+    public abstract void draw();
```

Comment:
this method better to move to the interface:
* interface - behavior
* abstract class - state

Reply:
It seems my brain was melted yesterday - done!